frontend/src/DeviceDetailsPage/TabPanels/DeviceDetailsNotesPanel.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Container } from "@material-ui/core";
+
+export default function DeviceDetailsNotesPanel(props) {
+  const { extras } = props;
+  return <Container>{extras}</Container>;
```

Comment:
could use some padding to read more easily but we've already said devicedetails page is gonna go through some redesigns so i'm not requesting a change xD

Reply:
tell that to the linter

Reply:
lsdjfalsdk i meant more like adding `padding=3em` or something so the frontend text is easier to read

Reply:
This isn't a finalized page, it's just a placeholder ATM so I don't think it's worth putting in the effort to make it nice since we're just going to scrap this anyways